### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by plehegar** [w3c/manual-of-style#11]
For a 1.1 version, we should recommend the following:

www.w3.org/TR/xxx11

This follows past conventions from HTML, CSS, and WCAG.

Version doc needs update as well:
 https://www.w3.org/2005/05/tr-versions#shortnames

cc @ashimura

**Comment by michael-n-cooper** [comment]
My preference is www.w3.org/TR/xxx-1.1

The alphabetic component should be lowercase, in keeping with the majority of English URIs.

The alphabetic component should be separated from the version component by a hyphen, just as it is separated from the date component in dated TR URIs.

The version number should include a dot to deliminate major and minor versions (if applicable), to disambiguate 1.1 (major version 1 minor version 1) from 11 (major version 11 minor version 0). When starting work on a spec that kind of thing seems unnecessary, but as a spec grows my experience is one wishes they had started this way from the beginning.

A version number without a dot in it (www.w3.org/TR/xxx-1) should refer to the latest minor version of the indicated major version. 

A shortname without a version number at all (www.w3.org/TR/xxx) should refer to the latest major version, latest minor version of that.

**Comment by swickr** [comment]
Speaking only for my personal opinion, I concur with Michael's preference for "xxx-1.1".

Furthermore, I'd leave to the judgement of each Working Group when to increment the major and minor version numbers.  There can be more considerations for specs than for software releases, so fixed semantics for versioning don't seem necessary to me.

I'd add this syntax to [Manual of Style](https://www.w3.org/Guide/manual-of-style/) as a recommendation and not a requirement.
